src/views/edition/components/imaged-objects.vue

```diff
@@ -21,25 +32,36 @@ import Vue from 'vue';
 import Waiting from '@/components/misc/Waiting.vue';
 import ImagedObjectCard from './imaged-object-card.vue';
 import { ImagedObject } from '@/models/imaged-object';
+import SearchBar from '@/components/search-bar.vue';
+import { SearchBarParams, SearchBarValue } from '@/components/search-bar.vue';
 
 export default Vue.extend({
     components: {
         ImagedObjectCard,
         Waiting,
+        SearchBar,
     },
     computed: {
         imagedObjects(): ImagedObject[] {
             return this.$state.imagedObjects.items;
-        }
+        },
+    },
+    methods: {
+        // onImagedObjectsSearch(event) {},
     },
     created() {
         this.$state.prepare.edition(this.$state.editions.current!.id);
-    }
+    },
 });
 </script>
 <style scoped>
-ul.list-unstyled {
-       height: calc(100vh - 123px);
-    overflow: auto;
+/* .card{
+    display: inline-block;
+    width: calc(25% - 20px);
+    margin: 10px;
+} */
+.direction {
+    float: right;
```

Comment:
I did not look too closely at what is going on here, but it seems to me that `display: inline-block;` is often a better choice than using `float` (though `float` may be the best this to use here, I didn't dig in deep enough to tell for sure).